app/jobs/reports/drop_off_report.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require 'reporting/drop_off_report'
+
+module Reports
+  class DropOffReport < BaseReport
+    REPORT_NAME = 'drop-off-report'
+
+    attr_accessor :report_date
+
+    def perform(report_date)
+      self.report_date = report_date
+
+      subject = "Drop Off Report - #{report_date.to_date}"
+      JSON.parse(configs).each do |config|
+        reports = [report_maker(config['issuers']).as_emailable_reports]
+        ReportMailer.tables_report(
```

Comment:
some of our emailable reports [allow multiple emails](https://github.com/18F/identity-idp/blob/main/app/jobs/reports/authentication_report.rb#L21) and issuers per configuration entry. seems likely that will be the case here, can we update the code to allow more than one partner email?